baker/DeployUtils.ts

```diff
@@ -27,16 +32,28 @@ const defaultCommitMessage = async (): Promise<string> => {
  */
 const bakeAndDeploy = async (
     message?: string,
-    email?: string,
-    name?: string
+    lightningQueue?: DeployChange[]
 ) => {
     message = message ?? (await defaultCommitMessage())
 
     const baker = new SiteBaker(BAKED_SITE_DIR, BAKED_BASE_URL)
 
     try {
-        await baker.bakeAll()
-        await baker.deployToNetlifyAndPushToGitPush(message, email, name)
+        if (lightningQueue?.length) {
+            for (const change of lightningQueue) {
+                // if you know how to close the connection without making
+                // subsequent runs fail, please do.
+                await db.getConnection()
```

Comment:
I think all we need is to call `dataSource.initialize()` before using the DB connection, which `getConnection()` does.
We should find a nice spot to do this.

Reply:
> I think all we need is to call dataSource.initialize() before using the DB connection, which getConnection() does.

Are you suggesting to replace the call to `getConnection()` with a call to `dataSource.initialize()`?

> We should find a nice spot to do this.

Agreed. I was trying to move it all the way up to the main script but found it in the end clearer for now to open up the connection closer to where it is used rather than obscuring the top script with an IIFE or some other way to `await` there. It somewhat feels inefficient, but I don't know enough about `initialize()` to know if that's actually true.

Reply:
> Are you suggesting to replace the call to getConnection() with a call to dataSource.initialize()?

That's one way to do it, and it's at least a bit clearer than what we have right now.
But ideally we really should pull it out further, in such a way that we never need to care whether the connection's been established already or not.

Reply:
ok, I've made a few updates that allow for opening and closing the db connection in the main script